### DISCUSSION TITLE
tar压缩文件错误问题

### DISCUSSION THREAD

**Discussion by JAINKRE** [1Panel-dev/1Panel#5082] [Q&A]
### 1Panel 版本

v1.10.2-lts

### 问题描述

使用面板tar压缩普通文件夹（24GB），操作日志显示成功后，下载本地解压，bandzip报错：
![20240324-201811](https://github.com/1Panel-dev/1Panel/assets/63288007/73b9e7c6-6a41-4630-bb2a-78853e917679)

发现解压时出现乱码文件夹，且该文件夹并不存在压缩对象中：
![PixPin_2024-03-24_20-19-42](https://github.com/1Panel-dev/1Panel/assets/63288007/1ef39937-57b5-4065-b6fc-5198213324c0)
> 执行压缩`1`文件夹为`tar`格式

起初怀疑是传输过程中小概率丢包问题，md5校验数据一致：
![PixPin_2024-03-24_20-10-48](https://github.com/1Panel-dev/1Panel/assets/63288007/29510a39-5d6b-40ce-b032-486d68c743b4)
> 服务器端校验

![PixPin_2024-03-24_20-11-53](https://github.com/1Panel-dev/1Panel/assets/63288007/c91235d9-5197-4d69-a07b-7e8e34c7d2de)
> 本地校验

于是，服务器端尝试解压`1.tar`，解压貌似成功（日志显示状态成功），没有发现报错提示，解压后文件夹路径正常进入，文件存在。

对`1`文件夹换压缩格式为zip，下载后尝试解压，成功。

### 重现步骤

* 文件 > 压缩文件夹为tar格式
* 下载 > bandzip尝试解压失败
* 测试> 压缩文件夹为zip格式，下载解压成功
* 说明> 压缩tar和zip日志均提示成功

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们现在本地环境复现一下上述问题。

**Comment by zhengkunwang223** [comment]
用 1Panel 压缩之后 你先在本机解压一下 看看正常吗

**Comment by JAINKRE** [comment]
> 用 1Panel 压缩之后 你先在本机解压一下 看看正常吗

1panel使用zip压缩下载本地解压正常
1panel使用tar压缩下载本地解压出现上述问题，但是在1panel面板中解压貌似正常，因为没有看到错误提示，而夹压后的文件夹包含众多子文件无法哈希校验
